src/foam/nanos/crunch/Capability.js

```diff
@@ -197,6 +197,13 @@ foam.CLASS({
       javaFactory: 'return foam.mlang.MLang.TRUE;',
       documentation: 'Predicate used to omit or include capabilities from capabilityDAO'
     },
+    {
+      class: 'foam.mlang.predicate.PredicateProperty',
+      name: 'capabilityVisibilityPredicate',
+      networkTransient: true,
```

Comment:
```suggestion
```

Based on our conversation with @jlhughes in Slack we shouldn't have this networkTransient

Reply:
yes, should not be networkTransient.